packages/react-server-cli/src/cli.js

```diff
@@ -1,5 +1,4 @@
 require("babel-core/register");
 
 const cli = require(".");
-
-cli.run(cli.parseCliArgs());
+cli.parseCliArgs().then(cli.run);
```

Comment:
Okay, so let me make sure I understand this: Because `sslize` is an _async_ function now it implicitly returns a promise, which we chain off of here?